organization/_posts/2017-10-05-startup.md

```diff
@@ -0,0 +1,90 @@
+
+#### *Present*: Dario Menasce, Benedikt Hegner, Daniel Elvira, Michel Jouvin, Peter Elmer, Graeme Stewart, John Harvey, Andrea Valassi, David Lange, Ian Bird, Mark Neubauer, Liz Sexton-Kennedy
+
+## General News
+
+- Workshop announcement: agreement on a common announcement by WLCG and HSF.
+- WLCG intention for WLCG workshop is focussing on HL-LHC matters. Will draft a proposal for the HSF main theme for the HSF workshop. Michel, Graeme and Benedikt will prepare a proposal until GDB next week.
+- J. Templon in charge of organizing the next IEEE Data Science meeting (Fall 2018) is thinking at organizing a session focused on HEP
+  - Meeting date should be added to the Community Calendar (29 October - 1 November 2018)
+
+
+## CWP
+
+-   #### General matters and Editorial Board
+    - Core editorial team dividing up the chapters and working towards having a substantial amount done by next week. 
+    - Will update “computing challenge” section with the latest estimates from ATLAS and CMS presented to the LHCC.
+    - Add finished documents to HSF website for Chinese collaborators
+      - Action on contribution editors to do this themselves or ask Benedkit, Graeme or Michel.
+    - Follow-up items from later discussion
+      - Encourage WG groups to make sublementary material for whitepapers available on arXiv
+      - Distribute (close to) finished whitepapers to general list
+
+
+-   #### Simulation
+    - Added more information from the G4 community, thus essentially completing the document.
+
+-   #### Machine Learning
+    - No news
+
+-   #### Software Development
+    - **Finished**
+
+-   #### Software Trigger and Event Reconstruction
+    - Incorporating last comments and converging soon
+
+-   #### Data Access, Organisation and Management
+    - Tidy-up done by Oliver
+    - Next meeting tomorrow to finalize the draft
+
+-   #### Data and Software Preservation
+    - No progress during last week, hope to do something for next week
+
+-   #### Data Analysis and Interpretation
+    - Happy with analysis section. Hoping to get a contribution from Kyle re. Interpretation to finish things soon. 
+    - Many details are in contributed white papers, which is why the document itself is quite short.
+      - Need to sort out how to handle the contributed white papers in the finalized CWP.
+      - Agreed to submit them to arXiv where they have substantial content. (Currently attached to the HSF web site.)
+
+-   #### Workflow and Resource Management
+    - No news
+
+-   #### Visualization
+    - No news
+
+-   #### Event/Data Processing Frameworks
+    - No progress
+
+-   #### Careers, Staffing and Training
+    - Basically done, but additional info, comments or feedback from wider audience welcome. Proposed to distribute it to general list soon
+
+-   #### Facilities and Distributed Computing
+    - Remaining comments are more for longer term follow up than would change the document as it is now. Ian is going to capture these.
```

Comment:
Suggest a nicer phrasing: "Remaining comments are more for longer term follow up and will not significantly change the document as it is now."